src/w3c/headers.js

```diff
@@ -249,6 +249,20 @@ export function run(conf) {
   if (conf.isRegular && !conf.shortName) {
     pub("error", "Missing required configuration: `shortName`");
   }
+  if (conf.testSuiteURI) {
+    const url = new URL(conf.testSuiteURI, document.location);
+    const { host, pathname } = url;
+    if (
+      host === "github.com" &&
+      pathname.startsWith("/w3c/web-platform-tests/")
+    ) {
+      const msg =
+        "Web Platform Tests have moved to a new Github Organization at https://github.com/web-platform-tests. " +
+        "Please update your [`testSuiteURI`](https://github.com/w3c/respec/wiki/testSuiteURI) to point to the " +
+        `new tests directory (e.g., https://github.com/web-platform-tests/${conf.shortName} ).`;
```

Comment:
note, need the space at the end of the URL, otherwise the markdown parser thinks ")." are part of the url.